js/style/style.js

```diff
@@ -731,7 +732,7 @@ Style.prototype = util.inherit(Evented, {
 
     // Callbacks from web workers
 
-    'get sprite json': function(params, callback) {
+    'get sprite json': function(_, params, callback) {
```

Comment:
I see. I'm ok with this. Let's rename `_` => `mapId` to reduce the mystery for future readers.